assets/training/model_management/src/azureml/model/mgmt/processors/factory.py

```diff
@@ -121,20 +122,29 @@ def create_mlflow_convertor(model_dir, output_dir, temp_dir, translate_params):
         raise Exception("Unsupported ASR model family")
 
 
-class DiffusersMLflowConvertorFactory(MLflowConvertorFactoryInterface):
-    """Factory class for diffusor model family."""
+class TextToImageMLflowConvertorFactory(MLflowConvertorFactoryInterface):
+    """Factory class for text to image model."""
+
+    STABLE_DIFFUSION_TASK_MAP = {
+        PyFuncSupportedTasks.TEXT_TO_IMAGE.value: StableDiffusionMlflowConvertor,
+        PyFuncSupportedTasks.TEXT_TO_IMAGE_INPAINTING.value: StableDiffusionInpaintingMlflowConvertor,
+    }
 
     def create_mlflow_convertor(model_dir, output_dir, temp_dir, translate_params):
         """Create MLflow convertor for diffusers."""
         misc = translate_params["misc"]
         if misc and SupportedTextToImageModelFamily.STABLE_DIFFUSION.value in misc:
```

Comment:
Why do we need this model_family parameter in misc ?

Reply:
for handling cases like, Diffusion pipeline and SD pipeline both support TEXT_TO_IMAGE Tasks. to differentiate between different model_families for the same task